rollup/rollup.config.ts

```diff
@@ -550,6 +552,26 @@ export default (args: Record<string, string>): rollup.RollupOptions[] => {
       assetFileNames: 'assets/[name][extname]',
       format: 'esm',
       dir: 'dist/main',
+      intro (chunk) {
+        // Ensure that all our files start with an import to `assets.ts`, as this
+        // file has a side-effect (setting monacoRequire on globalThis) that needs
+        // to be evaluated before VSCode evaluates. Because VSCode needs this side-effect
+        // for its own side-effects.
+        if (chunk.facadeModuleId == null) {
+          return ''
+        }
+
+        if (!chunk.facadeModuleId.startsWith(SRC_DIR)) {
+          return ''
+        }
+
+        let relativePath = path.relative(path.dirname(chunk.facadeModuleId), ASSETS_PATH)
+        if (!relativePath.startsWith('.')) {
+          relativePath = './' + relativePath
+        }
+
+        return `import "${relativePath}";\n`
+      },
```

Comment:
should be fixed by https://github.com/CodinGame/monaco-vscode-api/pull/241

Reply:
Thank you!! I was fighting for a while with this.